Added biweekly update for 4 - 18 March 2016

anelda opened the pull request:
@BinxiePeterson is my colleague at Talarify who will help me write updates for the next while

anelda committed:
Added biweekly update for 4 - 18 March 2016

gvwilson commented on _posts/2016/03/2016-03-18-weekly-review.md:
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+authors: ["Anelda van der Walt, Bianca Peterson"]
```
Please use `authors: ["Anelda van der Walt", "Bianca Peterson"]`, i.e., a list containing two strings.

anelda committed:
Update 2016-03-18-weekly-review.md

Changed author list from single string to list containing two strings and added some punctuation marks

anelda committed:
Added post for instructor training in South Africa
